tests/otel_integrations/test_openai.py

```diff
@@ -63,15 +198,15 @@ def request_handler(request: httpx.Request) -> httpx.Response:
                     cc_chunk.ChatCompletionChunk(
                         id='2',
                         choices=[
-                            cc_chunk.Choice(index=1, delta=cc_chunk.ChoiceDelta(content=' is secret', role='assistant'))
+                            cc_chunk.Choice(index=0, delta=cc_chunk.ChoiceDelta(content=' is secret', role='assistant'))
                         ],
                         created=1,
                         model='gpt-4',
                         object='chat.completion.chunk',
                     ),
                     cc_chunk.ChatCompletionChunk(
                         id='3',
-                        choices=[cc_chunk.Choice(index=2, delta=cc_chunk.ChoiceDelta(content=None, role='assistant'))],
+                        choices=[cc_chunk.Choice(index=0, delta=cc_chunk.ChoiceDelta(content=None, role='assistant'))],
```

Comment:
why did these change? why are both indices 0 now?

Reply:
These chunks are all part of the first item in the "choices" list, so should all have index 0.

```python
from openai import Client

response = Client().chat.completions.create(
    model="gpt-4",
    messages=[{"role": "user", "content": "Just say: The answer is secret."}],
    stream=True,
    # stream_options={"include_usage": True},
    # n=2,
)
for chunk in response:
    print(chunk.model_dump_json(exclude_none=True))

{"id":"chatcmpl-ASBkUgeHXBMarEwWnFLbT4vYMsldh","choices":[{"delta":{"content":"","role":"assistant"},"index":0}],"created":1731281710,"model":"gpt-4-0613","object":"chat.completion.chunk"}
{"id":"chatcmpl-ASBkUgeHXBMarEwWnFLbT4vYMsldh","choices":[{"delta":{"content":"The"},"index":0}],"created":1731281710,"model":"gpt-4-0613","object":"chat.completion.chunk"}
{"id":"chatcmpl-ASBkUgeHXBMarEwWnFLbT4vYMsldh","choices":[{"delta":{"content":" answer"},"index":0}],"created":1731281710,"model":"gpt-4-0613","object":"chat.completion.chunk"}
{"id":"chatcmpl-ASBkUgeHXBMarEwWnFLbT4vYMsldh","choices":[{"delta":{"content":" is"},"index":0}],"created":1731281710,"model":"gpt-4-0613","object":"chat.completion.chunk"}
{"id":"chatcmpl-ASBkUgeHXBMarEwWnFLbT4vYMsldh","choices":[{"delta":{"content":" secret"},"index":0}],"created":1731281710,"model":"gpt-4-0613","object":"chat.completion.chunk"}
{"id":"chatcmpl-ASBkUgeHXBMarEwWnFLbT4vYMsldh","choices":[{"delta":{"content":"."},"index":0}],"created":1731281710,"model":"gpt-4-0613","object":"chat.completion.chunk"}
{"id":"chatcmpl-ASBkUgeHXBMarEwWnFLbT4vYMsldh","choices":[{"delta":{},"finish_reason":"stop","index":0}],"created":1731281710,"model":"gpt-4-0613","object":"chat.completion.chunk"}
```